frontend/javascripts/oxalis/model/sagas/save_saga.ts

```diff
@@ -713,20 +714,22 @@ function* applyAndGetRevertingVolumeBatch(
   };
 }
 
-export function* pushTracingTypeAsync(
-  tracingType: "skeleton" | "volume",
+export function* pushSaveQueueAsync(
+  saveQueueType: SaveQueueType,
   tracingId: string,
+  isWkReady: boolean = false,
 ): Saga<void> {
-  yield* take("WK_READY");
-  yield* put(setLastSaveTimestampAction(tracingType, tracingId));
+  if (!isWkReady) yield* take("WK_READY");
```

Comment:
it seems a bit brittle to require from the caller to pass the correct `isWkReady` info. especially, since the parameter is optional. couldn't the saga check itself whether wk_ready was already dispatched?

Reply:
also see my next comment in line 994.